src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -34,7 +36,7 @@ public async Task EmptyFileShouldProduceDeclarationCompletions()
             const string expectedSetName = "declarations";
             var uri = DocumentUri.From($"/{this.TestContext!.TestName}");
 
-            using var client = await IntegrationTestHelper.StartServerWithText(string.Empty, uri);
+            using var client = await IntegrationTestHelper.StartServerWithTextAsync(string.Empty, uri);
```

Comment:
One of the new packages brought in some sort of threading analyzer which wanted to call things *Async if they are awaitable. I thought it made sense so didn't fight it too much.